### PR TITLE
added support for UD_MARATHI

### PR DESCRIPTION
**Pull request by polarlyset**

### CODE REVIEW

File: flair/datasets/treebanks.py [hunk 1]
```diff
@@ -1226,6 +1226,34 @@ def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
...
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        web_path = "https://raw.githubusercontent.com/UniversalDependencies/UD_English-EWT/master"
```
Comment (alanakbik) [inline flair/datasets/treebanks.py:1245]: These paths still point to the English UD treebank. They should point to the Marathi treebank.

**Review by alanakbik — CHANGES_REQUESTED** [review]
The object downloads the English UD treebank, but should download the Marathi one.

File: flair/datasets/treebanks.py (hunk 1 above)
Comment (polarlyset) [reply flair/datasets/treebanks.py]: Path and file names amended.

**Review by alanakbik — APPROVED** [review]

**Comment by alanakbik** [comment]
@polarlyset thanks for adding this!

**Pull request merged**